content/es-us/docs/latest/ASS_Tags.md

```diff
@@ -0,0 +1,816 @@
+Las coordenadas _x1_, _y1_, _x2_ y _y2_ se dan en el sistema de coordenadas de resolución del guión, como `\pos`. El subtítulo comienza en el punto (_x1_,_y1_) y se mueve con velocidad constante por lo que termina en (_x2_,_y2_).
+[Alineación]({{<relref path="ASS_Tags#\an">}}) influye en las coordenadas de movimiento de la misma manera que influye en las coordenadas `\pos`.
+
+En la segunda versión, los tiempos _t1_ y _t2_ se dan en milisegundos (milésimas de segundo) y son relativos a la hora de inicio del subtítulo. Por ejemplo, un valor _t1_ de 1500 significa que el movimiento comienza 1,5 segundos (un segundo y medio) después de que la línea haya aparecido en pantalla. Cuando especificas tiempos para el movimiento, la posición del subtítulo es la siguiente:
+
+1. Antes de _t1_, el subtítulo está fijado en el punto (_x1_,_y1_). 
+1. Entre _t1_ y _t2_, el subtítulo se mueve con velocidad constante desde (_x1_,_y1_) a (_x2_,_y2_). 
+1. Después de _t2_ el subtítulo está fijado en el punto (_x2_,_y2_).
+
+Tenga en cuenta que es permitido que _t1_ y _t2_ especifiquen tiempos mayores que la duración de la línea, pero puede que no sea muy útil hacerlo. Especificar _t1_ y _t2_ como 0 (cero) es lo mismo que usar la primera versión de
+`\ mover`; el movimiento se producirá desde el inicio de la línea hasta el fin de la línea.
+
+Hay algunas cosas que con **`\move` no se pueden hacer**:
+
+- No es posible realizar movimientos a velocidad no constante. El movimiento no puede, por ejemplo, comience lento y termine rápido.
+- Sólo puede haber una etiqueta o de posicionamiento o de movimiento en una línea. Poniendo ambos una `\pos` y una `\move` en una misma línea no funcionarán. Tampoco funciona colocar dos o más etiquetas `\move` en una sola línea.
+
+Si necesita hacer cualquiera de esos dos, deberá dividir el movimiento en segmentos que se realizan en líneas de subtítulos separadas. (Cómo hacer esto está fuera del alcance de esta página).
+
+{{<example-box>}}
+
+```plaintext
+\move(100,150,300,350)
+```
+
+Cuando aparece la línea en la pantalla, el subtítulo está en (100,150). Mientras se ve el subtítulo, este se mueve a velocidad constante de manera que llegará al punto (300,350) al mismo tiempo que desaparece.
+{{</example-box>}}
+{{<example-box>}}
+
+```plaintext
+\move(100,150,300,350,500,1500)
+```
+
+La línea aparece en (100,150). Después de que la línea haya aparecido durante medio segundo (500 milisegundos), comienza a moverse hacia (300,350), de modo que llegará al punto un segundo y medio (1500 milisegundos) después de que la línea apareció por primera vez en la pantalla.
+{{</example-box>}}
+
+{{<tag-def-box title="Origen de rotación" id="\org">}}\\org(<i>\<X></i>,<i>\<Y></i> ){{</tag-def-box>}}
+Establece el punto de origen utilizado para [rotación]({{<relref path="ASS_Tags#\frx">}}). Esto afecta a todas las rotaciones de la línea. Las coordenadas _X_ e _Y_ se proporcionan en píxeles de resolución de secuencia de comandos de números enteros.
```

Comment:
```suggestion
Establece el punto de origen utilizado para [rotación]({{<relref path="ASS_Tags#\frx">}}). Esto afecta a todas las rotaciones de la línea. Las coordenadas _X_ y _Y_ se proporcionan en píxeles de resolución de secuencia de comandos de números enteros.
```
To keep it consistent. There are other 3 instances where you use 'y' and not 'e'.